src/lib/client.ts

```diff
@@ -24,23 +22,23 @@ export function initializeClients() {
     account = privateKeyToAccount(`0x${privateKey}`);
 
     try {
```

Comment:
There is a redundancy here

You have thrown an error in a try block, and in the catch block, thrown the error again

If the createPublicClient or createWalletClient functions throw errors, they will automatically be propagated up the call stack without needing an explicit catch block.